src/captions/srt.rs

```diff
@@ -41,10 +39,10 @@ fn seconds_to_timestamp(milliseconds: i64) -> String {
     let d = Duration::milliseconds(milliseconds);
     format!(
         "{}:{}:{},{}",
-        d.whole_hours(),
-        d.whole_minutes() % 60,
-        d.whole_seconds() % 60,
-        d.whole_milliseconds() % 1000
+        format!("{:02}", d.whole_hours()),
```

Comment:
these formatters can go in the original string, like `"{:02}:{:02}:{},{}"`

Reply:
🤦  of course, talk about oversight